plugins/tracker-resources/src/components/CreateIssue.svelte

```diff
@@ -447,12 +463,68 @@
       }
     )
 
+    const messageData = await formTelegramMessage(value);
+    await sendTelegramNotification(messageData);
+
+    // todo Идеи для доработки сообщения:
+    // можно добавить приоритет
+    // + кнопку-обработчик, что задача взята в работу
+    // + пересылать в личные сообщения, а не в группу (если задача приватная) и тд.
+
+    // Также, наверное, можно использовать @hcengineering/telegram, но нужно больше информации, как работает плагин.
+
     draftController.remove()
     resetObject()
     descriptionBox?.removeDraft(false)
     isAssigneeTouched = false
   }
 
+  async function formTelegramMessage(issue: AttachedData<Issue>) {
```

Comment:
Better to do it on server using triggers, and provide custom model and server plugin.